docs/algorithm-registry.md

```diff
@@ -0,0 +1,17 @@
+# Algorithm Registry
+
+This file is designed to act as a source of truth regarding what signing
+algorithms are permitted across the Sigstore ecosystem. Any changes to this file
+**must** be reflected in the `SupportedAlgorithm` enumeration in
+[sigstore_common.proto](../protos/sigstore_common.proto).
+
+Refer to the [Sigstore: Configurable Crypto Algorithms](https://docs.google.com/document/d/18vTKFvTQdRt3OGz6Qd1xf04o-hugRYSup-1EAOWn7MQ/)
+specification for the design rationale for this registry.
+
+| Algorithm | Name | Usage |
+| --- | --- | --- |
+| ECDSA | ecdsa-sha2-256-nistp256 | sign/verify |
```

Comment:
Of course we'll be adding more in the future but it seems like we agree on these at the moment.